src/Components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -85,45 +86,44 @@ const AdvancedSearchModal: React.FC<IAdvancedSearchProps> = (props) => {
             styles={classNames.subComponentStyles.modal}
             layerProps={{ eventBubblingEnabled: true }}
         >
-            <BaseComponent theme={theme}>
-                <div className={classNames.header}>
-                    <div className={classNames.mainHeader}>
-                        <Icon
-                            iconName={'search'}
-                            styles={classNames.subComponentStyles.icon}
-                        />
-                        <h3 id={'titleId'} className={classNames.headerText}>
-                            {t('advancedSearch.modalTitle')}
-                        </h3>
-                    </div>
-                    <p className={classNames.subtitle}>
-                        {t('advancedSearch.modalSubtitle')}
-                    </p>
+            <div className={classNames.headerContainer}>
```

Comment:
if we are removing the base component, are we killing off the theme prop too?

Reply:
It should already be removed, check if you are seeing the latest changes